ui/src/app/views/signed-in/component.html

```diff
@@ -93,4 +93,6 @@ <h6 style="padding: 0.5rem;">
     <app-routing-spinner></app-routing-spinner>
   </div>
 </div>
-<app-bug-report></app-bug-report>
+<app-bug-report *ngIf="bugReportOpen"
```

Comment:
TOL: If we want to be able to launch this arbitrarily on any page, we could think about making it a more global piece of infrastructure at some point, e.g. rendering it at the top level of the app and driving it off some global state.

Reply:
I dunno if it is global state though because we want to open it conditionally with certain parameters when we trigger certain errors

Reply:
Driveby: What is TOL? (I'm assuming it's not Throne of Lies).

Also I think Brett means the global state would be "show or not show BugReport". The determination as to whether you toggle visibility could still be made at the component level (i.e. here). It feels like there's a good argument for making this component more of a singleton - the modal is already rendered in global scope, and you'd never want to open multiple separate bug report modals simultaneously.

Reply:
Heh, sorry. TOL means 'thinking out loud'. Terra started using it as a way to suggest longer-term ideas that are relevant but shouldn't necessarily be part of the current PR.

And yes, that was what I was thinking regarding the bug report modal. With that in place, any code path could just call e.g. `openBugReportModal('subject')` without having to manage its own instance of it.